### ISSUE TITLE
OpenMPI:A request was made to bind to that would result in binding more processes than cpus on a resource.

### ISSUE THREAD

**Issue by astrung** [open-mpi/ompi#626]
when i run a program what use MPI_Broadcast within a loop .when i use more processes,i get this error(with >=5 process)

```
WARNING: a request was made to bind a process. While the system
supports binding the process itself, at least one node does NOT
support binding memory to the process location.

  Node:  as-HP-Pro-2000-Microtower-PC

This usually is due to not having the required NUMA support installed
on the node. In some Linux distributions, the required support is
contained in the libnumactl and libnumactl-devel packages.
This is a warning only; your job will continue, though performance may be degraded.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
A request was made to bind to that would result in binding more
processes than cpus on a resource:

   Bind to:     NONE
   Node:        as-HP-Pro-2000-Microtower-PC
   #processes:  2
   #cpus:       1

You can override this protection by adding the "overload-allowed"
option to your binding directive.
```

But this program with run with 4 or lower process.
SO how to fix it??I use OpenMPI 1.8.4

**Comment by jsquyres** [comment]
You might want to upgrade to the recently-released 1.8.5; it has lots of bug fixes.

Additionally, you probably have 4 processors.  When you run with >= 5 MPI processes, MPI will emit this warning because you are oversubscribing the machine.

I have a dim recollection that we disabled binding by default when you run more processes than processors; can you try upgrading the 1.8.5 and see if the problem goes away?  (your performance will be degraded when you run more processes than processors, but I assume you're trying to do development / debugging runs on a small machine where the final performance doesn't matter)

**Comment by jsquyres** [comment]
Closing this due to lack of replies in about 3 weeks.  Please feel free to reply on this issue if you need further information.
